### PR TITLE
test suite

### PR DESCRIPTION
**Pull request by 0xaslan**
## Description 

Describe the changes or additions included in this PR.

## Test plan 

How did you test the new or updated feature?

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:

### CODE REVIEW

File: sdk/deepbook-v3/test/helper/constants.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+export class Constants {
+	public static readonly GAS_BUDGET = 6_000_000_000;
```
Comment (leecchh) [inline sdk/deepbook-v3/test/helper/constants.ts:5]: Is this arbitrary for now and will be changed later?

Comment (0xaslan) [reply sdk/deepbook-v3/test/helper/constants.ts:5]: This is the max gas budget to make sure all deployments and tests don't run out of gas.

File: sdk/deepbook-v3/test/helper/utils.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { GetCoinsParams, SuiClient } from '@mysten/sui/client';
+import { getFaucetHost, requestSuiFromFaucetV0 } from '@mysten/sui/faucet';
+import { Ed25519Keypair } from '@mysten/sui/keypairs/ed25519';
+
+import { RPC } from './constants';
+
+function base64ToUint8Array(base64: string) {
```
Comment (hayes-mysten) [inline sdk/deepbook-v3/test/helper/utils.ts:10]: @mysten/sui/utils exports this as `fromB64`

File: sdk/deepbook-v3/test/client.test.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+let toolbox!: TestToolbox;
+let coins: CoinMap;
+let deepbookPackageId: string;
+let deepbookRegistryId: string;
+let deepbookAdminCap: string;
+
+beforeAll(async () => {
+	toolbox = await setupSuiClient();
+	const tokenSourcesPath = path.join(__dirname, 'data/token');
```
Comment (hayes-mysten) [inline sdk/deepbook-v3/test/client.test.ts:20]: I'd probably wrap all this up as a method on toolbox so you don't have to repeat it in multiple test files

**Review by hayes-mysten — APPROVED** [review]

**Pull request merged**